config/CMakeLists.txt

```diff
@@ -50,16 +50,11 @@ if(JEMALLOC_FOUND)
   set(HAVE_JEMALLOC 1)
 endif()
 
-check_function_exists(lstat HAVE_LSTAT)
-if(NOT HAVE_LSTAT)
-  # os_unix.c uses lstat.c
-  message(SEND_ERROR "lstat() function not found on your system.")
-endif()
```

Comment:
`libuv` uses this internally... but I guess we can let it fail instead of dealing with it ourselves.